src/handlers/handler.start.ts

```diff
@@ -108,59 +74,30 @@ class StartCommandHandlerImpl extends StartCommandHandler {
    */
   async handleFoundPrompt(insertionRequest: InsertionRequestInfo, verbose: boolean): Promise<void> {
     try {
-      // Determine completion type: 'complete' or 'update'
-      const completionType = this.findCompletionType(
-        insertionRequest.fileContent,
-        insertionRequest.prompt
-      );
-      // Create a prompt context by extracting surrounding content
-      const promptArray = await userPromptService.findPromptContext(
-        insertionRequest.fileContent,
-        insertionRequest.prompt,
-        verbose
-      );
+      // TODO : Create prompt context using dependency graph.
```

Comment:
Coming soon?

Reply:
The embeddings part is still not done. Right now I used a graph ranking algorithm to find the most relevant node (file). A relevant node is basically a import statement to a file, this approach still sends a lot of code with prompts with embeddings we will be able to send only used part of every node. 
So I left this TODO to remind me from where to start the embeddings integrations.